api-schema/src/main/java/org/stellar/anchor/api/custody/fireblocks/AmlScreeningResult.java

```diff
@@ -0,0 +1,9 @@
+package org.stellar.anchor.api.custody.fireblocks;
+
+import lombok.Data;
+
+@Data
+public class AmlScreeningResult {
```

Comment:
Please comment on the acronym AML.

Reply:
AML is a fairly common acronym in finance for anti-money laundering.